src/main/java/de/dhbw/karlsruhe/students/mailflow/core/domain/email/Email.java

```diff
@@ -92,6 +93,10 @@ public Address getSender() {
     return this.emailMetadata.sender();
   }
 
+  public List<Header> getHeader() {
```

Comment:
```suggestion
  public List<Header> getHeaders() {
```

(in case you accept this suggestion, please also think about its usages :))